core/variant.cpp

```diff
@@ -1589,14 +1617,12 @@ String Variant::stringify(List<const void *> &stack) const {
 		case OBJECT: {
 
 			if (_get_obj().obj) {
-#ifdef DEBUG_ENABLED
```

Comment:
Shouldn't this `#ifdef` stay?

Reply:
No... stringify is not something that will affect performance anyway so it should be fine. The ifdefs were there because the reverse hash map in 3.x at some point was removed durin release build but then kept. Now it's gone so there is no longer a point.

Reply:
That's not the reason for my comment (the `#endif` ends anyway before the call to `to_string()`).

My point is that all other `if (ScriptDebugger::get_singleton() [...]` checks are inside an `#ifdef DEBUG_ENABLED`, because there's no point in non-debug builds. It's only for consistency.